lib/stat_tracker.rb

```diff
@@ -179,6 +180,45 @@ def win_percentage_per_team
     games_won_per_team.merge(total_games_per_team){|team_id, wins, games| (wins.to_f / games).round(2)}
   end
 
+  def games_won_per_team_for_season(season_id)
+    @game_teams.reduce(Hash.new(0)) do |result, game_team|
+      result[game_team.team_id] += 1 if game_team.season == season_id &&  game_team.result == "WIN"
+      result
+    end
+  end
```

Comment:
duplicate of method on line 113 (but different method name) We should delete this one. Depending on what the team decides we may actually change the method name to end with "for_season" instead of "per_season". The tests would need to be updated. As well as any methods dependent on it.